### PR TITLE
Initial Backend Implementation + Playback Experiment (P2P) success (2 people only)

### PR DESCRIPTION
**Pull request by xmquentin**
We got working P2P!

Finally, feels like it's been a while. The RSS will be valuable to us for ensuring we make our end of June deadline for milestone 2.

Note, for the experiment, I went ahead and added my own React FE, in a test `frontend/` folder. We will later adapt to the `roomz-frontend` repo. I encourage you both to read through this code, and have it make sense to you. I added a lot of comments on what's actually happening in laymen terms. Please ask questions about webrtc stuff.

Up next is taking on the Grid/Video Components, and ensure I can populate a grid of video screens cleanly for my test.

### CODE REVIEW

File: frontend/src/containers/App.js
Comment (ABFranco) [inline frontend/src/containers/App.js:17]: hehe

**Review by ABFranco — APPROVED** [review]
I don't have much to contribute to backend-review as usual. Frontend I guess I'll wait until we merge it into the main RFE repo. Looks good tho

**Comment by xmquentin** [comment]
> I don't have much to contribute to backend-review as usual. Frontend I guess I'll wait until we merge it into the main RFE repo. Looks good tho

**Pull request merged**
